data-types/4-python-tabular-data.py

```diff
@@ -0,0 +1,449 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.4.2
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# <img style="float: left;" src="../earth-lab-logo-rgb.png" width="150" height="150">
+#
+# # Earth Data Science Corps Summer 2020
+#
+# ![Colored Bar](../colored-bar.png)
```

Comment:
@nkorinek as you are creating the lessons notice that i've moved the images into the main directory so we don't need to copy them each time. can you please do that on your lessons as well?